frontend/rolecall/src/app/services/super-display.service.ts

```diff
@@ -0,0 +1,30 @@
+import {Injectable} from '@angular/core';
+
+
+@Injectable({providedIn: 'root'})
+export class DisplayService {
+  superIsOpenMap = new Map<string, boolean>();
+
+  delete(uuid: string) {
+    if (this.superIsOpenMap.has(uuid)) {    
+      this.superIsOpenMap.delete(uuid);
+    }    
+  }
+
+  isOpen(uuid: string): boolean {
+    if (!this.superIsOpenMap.has(uuid)) {    
```

Comment:
You can optionally simplify this to:
```
isSuperBalletOpen(balletId: string): boolean {
  return Boolean(this.superBalletIsOpenMap.get(balletId);
}
```
It'll handle both missing and false values. But you can leave the conditions separated if you want to be more explicit.

Reply:
Done.